packages/deployment/app/config.yml

```diff
@@ -13,6 +13,13 @@ default:
   message_adapter_repo: cumulus-message-adapter
   message_adapter_filename: cumulus-message-adapter.zip
 
+  customCompilation: true
+  injectWorkflowLambdaAliases: true
+
+  DLQDefaultTimeout: 60
+  DLQDefaultMessageRetentionPeriod: 1209600
```

Comment:
Just like in the other PR review, comment that this is 14 days or whatever.

Reply:
@laurenfrederick  sorry, I should probably have made this a clean branch: 

https://github.com/nasa/cumulus/pull/662#issuecomment-434445610